vsts-extension-google-play.json

```diff
@@ -2,7 +2,7 @@
     "manifestVersion": 1.0,
     "id": "google-play",
     "name": "Google Play",
-    "version": "3.164.1",
+    "version": "1.168.0",
```

Comment:
Why was the major version downgraded here?

Reply:
So what should the version be now here?